syntax = 'proto3';

package messages;

message DBotUtterance {
  string utterance = 1;
}
